src/SIM/SIMbase.C

```diff
@@ -1625,34 +1625,60 @@ bool SIMbase::project (Matrix& ssol, const Vector& psol,
 }
 
 
-bool SIMbase::project (Vector& values, const RealFunc* f,
-                       int basis, int iField, int nFields, double time) const
+bool SIMbase::project (Vector& values, const FunctionBase* f,
+                       int basis, int iField, int nFields,
+                       SIMoptions::ProjectionMethod pMethod, double time) const
 {
   bool ok = true;
+  values.resize(f->dim()*mySam->getNoNodes());
```

Comment:
the addition of this resize call breaks the stokes and navier-stokes regression tests. the vector should be of total model size (mySam->getNoDOFs()), not f->dim*mySam->getNoNodes(). the vector already has the correct size set by the caller in existing code, if you need this for the l2 stuff you have to resize it up front or fix the size you set.